templates/form-fields/file-field.php

```diff
@@ -51,7 +51,7 @@
 	  endif;
   endif; ?>
 </div>
-<input type="file" class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>" data-file_types="<?php echo esc_attr( implode( '|', $allowed_mime_types ) ); ?>" <?php if ( ! empty( $data->multiple ) ) echo 'multiple'; ?> name="<?php echo esc_attr( isset( $data->name ) ? $data->name : $data->key ); ?><?php if ( ! empty( $data->multiple ) ) echo '[]'; ?>" id="<?php echo esc_attr( $data->key ); ?>" placeholder="<?php echo empty( $data->placeholder ) ? '' : esc_attr( $data->placeholder ); ?>" />
+<input type="file" class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>" data-file_types="<?php echo esc_attr( implode( '|', $allowed_mime_types ) ); ?>" <?php if ( ! empty( $data->multiple ) ) echo 'multiple'; ?> name="<?php echo esc_attr( isset( $data->name ) ? $data->name : $data->key ); ?><?php if ( ! empty( $data->multiple ) ) echo '[]'; ?>" id="<?php echo esc_attr( $data->key ); ?>" placeholder="<?php echo empty( $data->placeholder ) ? '' : esc_attr( $data->placeholder ); ?>" data-file-key="<?php echo esc_attr( $data->key ); ?>" data-file-action="wpum_upload_profile_image"  data-file-nonce="<?php echo wp_create_nonce( 'verify_account_form' ); ?>" data-file-layout="<?php echo $data->key == 'user_avatar' ? 'compact circle' : ''; ?>" data-file-width="<?php echo $data->key == 'user_avatar' ? '250' : ''; ?>" data-file-height="<?php echo $data->key == 'user_avatar' ? '250' : ''; ?>" data-file-poster="<?php echo $data->value; ?>" data-file-posterkey="<?php echo $data->key == 'user_avatar' ? 'current_user_avatar_path' : 'user_cover_path'; ?>" />
```

Comment:
Getting this when visiting a registration form with a file field on it

 PHP Notice:  Undefined property: stdClass::$value in /Users/igp/sites/wpum/public/content/plugins/wp-user-manager/templates/form-fields/file-field.php on line 54

Reply:
@polevaultweb 
I've committed new changes for this.